.github/workflows/cd.yaml

```diff
@@ -0,0 +1,24 @@
+name: CD
+
+on:  # yamllint disable-line rule:truthy
+  create:
+    tags: ["v?[0-9]+.[0-9]+.[0-9]+"]
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout Code Repository
+        uses: actions/checkout@v2
```

Comment:
Quick question: Is this checking out the tag being created?  Or is it checking out `master`?  One possible issue with checking out `master` is that we could create a new tag and then immediately afterward merge another change into `master` and if this command is checking out `master`, it would include an extra commit not found in the tag.

Reply:
checkout would get the commit triggering the job, in this case the tagged release.